example/versions.tf

```diff
@@ -3,7 +3,10 @@ terraform {
   required_providers {
     equinix = {
       source  = "equinix/equinix"
-      version = "= 1.1.0"
+      version = "~> 1.14"
     }
   }
+  provider_meta "equinix" {
```

Comment:
@ocobleseqx suggested that we should not include `provider_meta` in example modules, and I'm on board with that; let's remove this.

Reply:
updated